src/com/cybozu/labs/langdetect/Command.java

```diff
@@ -134,6 +134,15 @@ public void generateProfile() {
             FileOutputStream os = null;
             try {
                 LangProfile profile = GenProfile.loadFromWikipediaAbstract(lang, file);
+                if(get("update") != null){
+                    try {
+                        FileInputStream is = new FileInputStream(file);
+                        LangProfile old_profile = JSON.decode(is, LangProfile.class);
+                        profile.merge(old_profile);
+                    }catch(Exception e){}
+                }else {
+                    profile.omitLessFreq();

```

Comment:
Sigh, that was probably reintroduced from a ctrl-z. Should definitely be removed.